test/externalTests/statistics.js

```diff
@@ -0,0 +1,7 @@
+const assert = require('assert')
+
+module.exports = () => async (bot) => {
+  const stats = await bot.requestStatistics()
+
+  assert.strictEqual(typeof stats, 'object')
```

Comment:
definitely going to need more testing then that stats is an object

Reply:
can u help me lol

Reply:
Check the values